Makefile

```diff
@@ -31,8 +31,10 @@ IMG_REPO ?= model-registry/server
 # container image build path
 BUILD_PATH ?= .
 # container image
-ifdef IMG_REGISTRY
-    IMG := ${IMG_REGISTRY}/${IMG_ORG}/${IMG_REPO}
+ifdef IMG
+	IMG := ${IMG}
+else ifdef IMG_REGISTRY
+    IMG := ${IMG_REGISTRY}/${IMG_ORG}/${IMG_REPO}:${IMG_VERSION}
```

Comment:
to me this is a great catch, if IMG was defined the makefile would have ignored previously

cc @Al-Pragliola  (if we want to do differently I'm also open to followups)